app/src/main/java/br/com/dionataferraz/vendas/HomeActivity.kt

```diff
@@ -50,33 +52,32 @@ class HomeActivity : AppCompatActivity() {
             startActivity(intent)
         }
     }
-
-    private fun setSaldo() {
-        val account = databaseAcc.AccDao().getAccount(1)
-        binding.tvAccountBalance.text = "R$ " + account.value.toString()
-    }
-
-    private fun configureAcc() {
-        val account = databaseAcc.AccDao().getAccount(1)
-
-        if (account == null) {
-            val dao = databaseAcc.AccDao()
-            val acc = AccountEntity(
+    fun getAccValues(): AccountEntity {
+        val account = databaseAcc.AccDao().getAccount()
+        if (account.isEmpty()) {
+            val newAcc = AccountEntity(
                 id = 1,
-                value = 300.00,
-                date = Date(2021, 12, 5, 10, 0),
-                type = Operation.DEPOSIT
+                value = 0.0,
+                date = Date()
             )
-
-            dao.insertAccount(acc)
+            databaseAcc.AccDao().insertAccount(newAcc)
         }
+        return databaseAcc.AccDao().getAccount().get(0)
+    }
 
+    @SuppressLint("SetTextI18n")
+    private fun setViewValues() {
+        val account = getAccValues()
+        val user = databaseUser.DAO().getUser().get(0)
```

Comment:
Nesse caso aqui pode ocorrer um erro caso não tenha algum usuário, para evitar que a app quebre nós podemos verificar se a lista não está vazia

Reply:
Outra sugestão é que podemos utilizar first, igual pode ocorrer o problema descrito acima, mas aqui ele lança uma `Exception` informando que a lista está vazia
```suggestion
        val user = databaseUser.DAO().getUser().first()
```